cmd/build/v2/services_test.go

```diff
@@ -27,7 +28,9 @@ func TestAllServicesAlreadyBuilt(t *testing.T) {
 	fakeReg := test.NewFakeOktetoRegistry(nil)
 	bc := NewBuilder(nil, fakeReg)
 	alreadyBuilt := []string{}
-	fakeReg.AddImageByName(alreadyBuilt...)
+	if err := fakeReg.AddImageByName(alreadyBuilt...); err != nil {
+		oktetoLog.Infof("error adding image to registry: %s", err)
+	}
```

Comment:
shouldn't we fail the test here? We can user `require.NoErr`

Reply:
Sounds good. Should I use the require package for err handling in all the other test files too?

Reply:
@abs007 Yes please :pray:

Reply:
Will do then